Description: Preseeding fails to parse when dbc should not handle database
 If the user sets <package>/dbconfig-install to false the other parameters
 are not processed. By removing on early return check, these parameters do
 get parsed.
Author: Paul Gevers <elbrus@debian.org>

diff --git a/dpkg/common b/dpkg/common
index 1d8e31e..5da32af 100644
--- a/dpkg/common
+++ b/dpkg/common
@@ -343,9 +343,6 @@ dbc_read_package_debconf(){
 	db_get $dbc_package/dbconfig-upgrade && dbc_upgrade="$RET"
 	db_get $dbc_package/dbconfig-remove && dbc_remove="$RET"
 
-	# if they don't want help, then we're done and return early
-	if [ "$dbc_install" != "true" ]; then return 0; fi
-	
 	# otherwise, just to make sure dbtype is set...
 	_dbc_sanity_check dbtype || dbc_install_error
 
